appveyor.yml

```diff
@@ -67,6 +67,7 @@ for:
      && git config --global user.name "AppVeyor"
      && git add download\%RINGMESH_PACKAGE%.zip
      && git commit -m "Update Windows master release"
+     && git pull origin master

```

Comment:
Why is this pull necessary? Does the origin master change in the meantime?

Reply:
After the last PR was merged, AppVeyor faildes with this error :

 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'https://BotellaA:*****@github.com/ringmesh/web'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.